plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -73,9 +73,11 @@ export class EventsProcessor {
             if (!team) {
                 throw new Error(`No team found with ID ${teamId}. Can't ingest event.`)
             }
-
             if (data['event'] === '$snapshot') {
                 if (team.session_recording_opt_in) {
+                    if (await this.teamManager.isAboveUsageLimit(team.organization_id, 'recordings')) {
+                        return null
```

Comment:
Wonder if we should temporarily just log something instead of actually dropping for now. That way we can check out the logs and be sure we are not doing anything terrible.

Also wondering if we can add this to statsd or something to track the number of droppages.

The potential negative impact of getting this wrong is very high so I think we should proceed cautiously...